src/systems/sprites.js

```diff
@@ -137,15 +110,15 @@ const raycastOnSprite = (function() {
   };
 })();
 
-function getHoverableVisuals(el) {
+const getHoverableVisuals = el => {
```

Comment:
Doesn't actually matter but is there a reason you changed these? I tend to only use arrow functions when I need to actually bind `this` or use implicit return. No strong preference but we should probably come up with a rule for code consistency.

Reply:
Just for the consistency in the file (although still some `function`s are remained as inner functions). Yes, it's good if we have a rule.